phpseclib/File/ANSI.php

```diff
@@ -213,6 +181,16 @@ class File_ANSI
      */
     function File_ANSI()
     {
+        $attr_cell = new stdClass();
+        $attr_cell->bold = false;
+        $attr_cell->underline = false;
+        $attr_cell->blink = false;
+        $attr_cell->background = 'black';
+        $attr_cell->foreground = 'white';
+        $attr_cell->reverse = false;
+        $this->base_attr_cell = clone($attr_cell);
```

Comment:
Yup - but it's one that can be remedied with PHP_Compat:

https://github.com/pear/PHP_Compat/blob/master/Compat/Function/clone.php

In the 2.0 branch I did `clone $attr_cell` instead of `clone($attr_cell)` since I think that looks better.